test/library/packages/LinearAlgebra/wongalvis14/inv-test.chpl

```diff
@@ -0,0 +1,16 @@
+use LinearAlgebra;
```

Comment:
Can you put this test in `test/library/packages/LinearAlgebra/correctness/` instead?

Reply:
Soon (or maybe not so soon) it may include performance testing as well like LU, so that's why I put it in this folder

Reply:
For now I've moved it, I think in the future when there's performance testing they'll be further modified. Thanks.